src/journal/journald-server.c

```diff
@@ -326,6 +326,10 @@ static int system_journal_open(Server *s, bool flush_requested) {
                  * Perform an implicit flush to var, leaving the runtime
                  * journal closed, now that the system journal is back.
                  */
+
+                 FIXME: is this still desired if we fail to open the
+                 persistent journal ?
```

Comment:
I must admit I haven't looked at this part of this code (the call to server_flush_to_var() for dealing with a ENOSPC recover). But it just seemd weird that we tried to flush whereas we failed to open the persistent journal.

Anyways, I'll try to look into this when I'll get more time to spend on it.
allow.